unit-tests/src/test/java/org/eclipse/collections/impl/AbstractRichIterableTestCase.java

```diff
@@ -1329,6 +1329,16 @@ public void toSortedMap_with_comparator()
         Verify.assertListsEqual(Lists.mutable.with(3, 2, 1), map.keySet().toList());
     }
 
+    @Test
+    public void toSortedMap_with_function()
```

Comment:
I still see this naming convention (i.e. toSortedMap_with_function) in number of test classes. Please kindly fix all of them.

Reply:
Sorry, missed them. Now fixed.